src/components/Canvas/CanvasTemplate.vue

```diff
@@ -280,35 +421,50 @@ export default {
         width: this.pageWidth + "px",
       };
     },
+    getOpacity() {
+      return {
+        last_block:
+          this.imageBlockController.length > 1 &&
+          !this.initialImageFocus.hasFocus,
+      };
+    },
   },
   created() {
     const { width, height } = useWindowSize();
     this.windowHeight = height;
     this.windowWidth = width;
 
-    this.nextPositions = this.potentialPositions
-      .sort(() => Math.random() - 0.5)
-      .slice(0, 3);
-
     // Default page size set to two times the current windows size
     this.pageHeight = 2 * this.windowHeight;
     this.pageWidth = 2 * this.windowWidth;
 
     const factor = getFactor(this.windowHeight, this.windowWidth);
 
     // Find the middle of the page to insert the first image
-    this.centralImageTopPosition =
+    const firstBlockCentralImageTopPosition =
       this.pageHeight / 2 - thumbnailHeight(factor) / 2;
-    this.centralImageLeftPosition =
+    const firstBlockCentralImageLeftPosition =
       this.pageWidth / 2 - thumbnailWidth(factor) / 2;
 
-    this.currentXPosition = this.centralImageLeftPosition;
-    this.currentYPosition = this.centralImageTopPosition;
+    this.insertBlock(
+      this.initialImageId,
+      firstBlockCentralImageTopPosition,
+      firstBlockCentralImageLeftPosition,
+      0,
+      this.relatedImages[this.initialImageId]
+    );
+    this.imageBlockController[0].oldCentralImageTag = this.initialCentralTag;
```

Comment:
why not renaming `oldCentralImageTag`to `tag`?

Reply:
The oldCentralImageTag is used to store the tag of the previous block to be able to display it in the central image when the block disappear (shift of the array). This is to preserve the central image when the old block is removed. 
In the context of this line, this is used like that because it's the first usage of the oldCentralTag and the value of the block is used to manage this specific case.